prophecypracticum/tests/test_prophecy.py

```diff
@@ -60,6 +60,14 @@ def test_set_feedback_rating():
     my_prophecy.set_feedback_rating(4)
     assert my_prophecy.feedback_rating == 4
 
+def test_change_feedback_rating():
+    """The supplicant should be able to change a feedback rating."""
+    my_prophecy = prophecy.Prophecy(test_datetime, prophecy_text="You change your feedback today.")
+    my_prophecy.set_feedback_rating(4)
+    old_rating = my_prophecy.get_feedback_rating()
+    assert old_rating == 4
+    my_prophecy.set_feedback_rating(2)
+    assert my_prophecy.get_feedback_rating() == 2 and my_prophecy.get_feedback_rating() != old_rating
```

Comment:
I don't think we need to check that it's not the same as the old rating unless we're going to prevent that in the code. I think it makes things unnecessarily complicated.

Reply:
I thought the same, but I just threw it in anyhow.

Reply:
Ok, if you remove it and push again, then we can merge that.

Reply:
Okay